cmd/root.go

```diff
@@ -98,3 +99,9 @@ func readStdin() (string, error) {
 	str, err := bufio.NewReader(os.Stdin).ReadString('\n')
 	return strings.TrimSpace(str), err
 }
+
+func noCommand(cmd *cobra.Command, args []string) {
+	fmt.Println("No command specified.")
+	cmd.Help()
+	os.Exit(1)
+}
```

Comment:
Could you replace `Run` with `RunE` like below?
https://github.com/kubernetes-sigs/kind/blob/v0.20.0/pkg/cmd/kind/create/create.go#L37-L43

Reply:
@dongrie Excuse me, but I have changed my opinion.

If `RunE` returns a non-nil value, the program will panic in the end, right?
If so, it's better choice to call `os.Exit(1)` in `Run` in this case.

Reply:
```golang
RunE:  noCommand,

func noCommand(cmd *cobra.Command, args []string) error {
	cmd.Help()
	return errors.New("no command specified - use RunE")
}
```
@siburu I experimented with a code like this
Got the same result.
<img width="637" alt="Screenshot 2023-09-06 at 9 39 21" src="https://github.com/hyperledger-labs/yui-relayer/assets/126585618/b5e33f3f-d0cc-4b4e-97ef-1d2db85df756">

Reply:
@dongrie Thank you very much!
I thought that `log.Fatal` caused panics, but I was wrong.
Well, please replace `Run` with `RunE` :bow:

Reply:
https://github.com/hyperledger-labs/yui-relayer/pull/104/commits/62c06373015fda67c2edc2a910336cbeee919376
Change to `RunE`